src/main/webapp/app/entities/lecture/lecture-attachments.component.ts

```diff
@@ -59,26 +59,27 @@ export class LectureAttachmentsComponent implements OnInit {
 
     saveAttachment() {
         if (!this.attachmentToBeCreated) {
+            // TODO: Fix this Null Pointer, maybe call addAttachment()?
```

Comment:
@maximilianmeier Can you look at this Null Pointer?

Reply:
done